test/e2e/adminapi_redeployvm.go

```diff
@@ -49,7 +45,7 @@ var _ = XDescribe("[Admin API] VM redeploy action", func() {
 		Expect(resp.StatusCode).To(Equal(http.StatusOK))
 
 		By("verifying through Azure activity logs that the redeployment happened")
-		err = wait.PollImmediate(10*time.Second, 5*time.Minute, func() (bool, error) {
+		err = wait.PollImmediate(10*time.Second, 10*time.Minute, func() (bool, error) {
```

Comment:
Why do we increase timeout? IIRC 5 minutes worked fine untill it got broken for unrelated reason.

Reply:
I have seen most cases of redeploy actions taking between 4m30s and 5m30s, this should reduce the false-failures, while still alerting to an actual fault.

Reply:
@drewandersonnz could you please give few links to the exampels where in CI failed due to exiding 5 minutes timeout?

Reply:
@m1kola seems I can't, all events in a recent search are with 3m30s and 4m30s, it may have been a data ingest and availability delay in the ActivityLog ausing one of the failures I saw. 

Do you want me to revert this 10m change?

Reply:
@drewandersonnz I suggest that we revert back to 5 minutes and decide whether it is worth increasing timeout once we start seen failures due to timeouts.

Reply:
Reverted back to 5 minutes.